ckanext/switzerland/helpers.py

```diff
@@ -362,3 +363,44 @@ def uri_to_iri(uri):
         return iri
     except:
         raise ValueError("Provided URI can't be converted to IRI")
+
+def get_timedelta_by_frequency(accrual_periodicity):
+
+    time_deltas = {
+        'http://purl.org/cld/freq/completelyIrregular': None,  # noqa
+        'http://purl.org/cld/freq/continuous': None,  # noqa
+        'http://purl.org/cld/freq/daily': timedelta(days=1),  # noqa
+        'http://purl.org/cld/freq/threeTimesAWeek': timedelta(days=2),  # noqa
+        'http://purl.org/cld/freq/semiweekly': timedelta(days=4),  # noqa
+        'http://purl.org/cld/freq/weekly': timedelta(days=7),  # noqa
+        'http://purl.org/cld/freq/threeTimesAMonth': timedelta(days=10),  # noqa
+        'http://purl.org/cld/freq/biweekly': timedelta(days=14),  # noqa
+        'http://purl.org/cld/freq/semimonthly': timedelta(days=15),  # noqa
+        'http://purl.org/cld/freq/monthly': timedelta(days=30),  # noqa
+        'http://purl.org/cld/freq/bimonthly': timedelta(days=60),  # noqa
+        'http://purl.org/cld/freq/quarterly': timedelta(days=91),  # noqa
+        'http://purl.org/cld/freq/threeTimesAYear': timedelta(days=122),  # noqa
+        'http://purl.org/cld/freq/semiannual': timedelta(days=183),  # noqa
+        'http://purl.org/cld/freq/annual': timedelta(days=365),  # noqa
+        'http://purl.org/cld/freq/biennial': timedelta(days=730),  # noqa
+        'http://purl.org/cld/freq/triennial': timedelta(days=1095),  # noqa
```

Comment:
I quickly looked for alternatives to timedelta and found [dateutil.relativedelta](https://dateutil.readthedocs.io/en/latest/relativedelta.html), which offers a more precise solution to our problem. E.g. the monthly delta is now set to 30 days, while we actually want 1 month (ranging from 28-31 days). Maybe check this example: https://stackoverflow.com/questions/546321/how-do-i-calculate-the-date-six-months-from-the-current-date-using-the-datetime/4406260#4406260